jung-visualization/src/main/java/edu/uci/ics/jung/visualization/spatial/SpatialQuadTree.java

```diff
@@ -541,15 +539,13 @@ public void update(N node, Point location) {
   }
 
   @Override
-  public void layoutChanged(LayoutEvent<N> evt) {
-    Point location = evt.getLocation();
-    N node = evt.getNode();
-    this.update(node, evt.getLocation());
+  public void layoutNodePositionChanged(LayoutNodePositionChange.Event<N> evt) {
+    this.update(evt.node, evt.location);
   }
 
   @Override
-  public void layoutChanged(LayoutNetworkEvent<N> evt) {
-    this.update(evt.getNode(), evt.getLocation());
+  public void layoutNodePositionChanged(LayoutNodePositionChange.NetworkEvent<N> evt) {
+    this.update(evt.node, evt.location);
   }
```

Comment:
this event allows the update of one element in the tree (vice rebuilding the whole thing)